core/bug_api.php

```diff
@@ -508,6 +508,16 @@ public function validate( $p_update_extended = true ) {
 			category_ensure_exists( $this->category_id );
 		}
 
+		if ( is_blank( $this->fixed_in_version ) && config_get( 'bug_fixed_in_version_required', null, null, $this->project_id ) != OFF ) {
+			$t_resolved = config_get( 'bug_resolved_status_threshold', null, null, $this->project_id );
+			$t_closed = config_get( 'bug_closed_status_threshold', null, null, $this->project_id );
+			$t_resolution_fixed = config_get( 'bug_resolution_fixed_threshold', null, null, $this->project_id );
+			if ( $this->status >= $t_resolved && ( $this->status < $t_closed || !$t_bug_resolution_is_fixed ) ) {
```

Comment:
$t_bug_resolution_is_fixed variable is not defined - did you mean $t_resolution_fixed ?.